core/src/main/java/org/radargun/stats/MultiStatistics.java

```diff
@@ -0,0 +1,118 @@
+package org.radargun.stats;
+
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.radargun.Operation;
+
+/**
+ * @author Radim Vansa &lt;rvansa@redhat.com&gt;
+ */
+public abstract class MultiStatistics implements Statistics {
```

Comment:
It's more about a separation of concerns. MultiStatistics does multiplex all the calls, while CommonStatistics is kind of template for MultiStatistics - it defines concrete mix of stats.